README.md

```diff
@@ -123,6 +123,6 @@ In order to produce a uniquely identifiable distribution:
 Feedstock Maintainers
 =====================
 
-* [@brandon-rhodes](https://github.com/brandon-rhodes/)
+* [@brandon-rhodes ReimarBauer](https://github.com/brandon-rhodes ReimarBauer/)
```

Comment:
This looks malformed?

Reply:
This was in the old commit the recent has only these changes


![Bildschirmfoto vom 2020-01-14 18-21-03](https://user-images.githubusercontent.com/278728/72366536-bd11b680-36fa-11ea-9015-83c9be93b18b.png)